main/inc/lib/template.lib.php

```diff
@@ -553,22 +553,17 @@ public function set_js_files()
         global $disable_js_and_css_files, $htmlHeadXtra;
 
         //JS files
```

Comment:
Add a new condition.
If "chat" or $_configuration['my_courses_view_by_session'] are turn "on" then add the cookie library.